static/scss/profile.scss

```diff
@@ -32,6 +32,9 @@
     flex-direction: row;
     color: $lightgrey !important;
     text-transform: uppercase;
+    position: absolute;
+    right: 0;
+    padding-right: 50px;
```

Comment:
Can you set the font for `.profile-card-header > span` to be 18px or put `.title` on the `<span>` tags in `.profile-card-header` on the education and work history elements?